website/integrations/services/owncloud/index.md

```diff
@@ -0,0 +1,229 @@
+        -   Redirect URI: `oc://android.owncloud.com`
+-   iOS Application:
+    -   Protocol Settings:
+        -   Client ID: use the pre-defined values from [Owncloud](https://doc.owncloud.com/server/10.15/admin_manual/configuration/user/oidc/oidc.html#client-id)
+        -   Client Secret: use the pre-defined values from [Owncloud](https://doc.owncloud.com/server/10.15/admin_manual/configuration/user/oidc/oidc.html#client-secret)
+        -   Redirect URI: `oc://ios.owncloud.com`
+
+##### Applications
+
+#### Owncloud
+
+In the authentik Admin Interface, go to _Applications_ -> _Applications_, and create a new Application with the
+following settings for each of the providers defined above:
+
+-   Name: owncloud, owncloud-desktop, owncloud-android, or owncloud-ios
+-   Slug: same as name
+-   Provider: one of the providers you created in the previous section
+-   UI Settings:
+    -   Launch URL: You can set this to `blank://blank` to prevent the application from being listed on the authentik
+        home page. This may be useful for the desktop, android and ios applications, since you will not be able
+        to navigate directly to those applications.
+        :::note
+        Sometimes this field glitches out when creating the application, you may need to save the application and then
+        edit it to fill this field.
+        :::
+
+##### Service Discovery
+
+In order for the Owncloud Applications to be able to login via OIDC, your reverse proxy will have to be configured to rewrite
+
+`https://cloud.example.com/.well-known/openid-configuration`
+
+to
+
+`https://cloud.example.com/index.php/apps/openidconnect/config`
+
+The Owncloud [documentation](https://doc.owncloud.com/server/10.15/admin_manual/configuration/user/oidc/oidc.html#set-up-service-discovery)
+includes an example for accomplishing this with Apache.
+
+In Traefik, use the [replacepathregex](https://doc.traefik.io/traefik/middlewares/http/replacepathregex/) middleware
+to accomplish the same thing.
+When using Traefik in docker-compose, add the following labels to your container:
+
+```yaml
+labels:
+  ... # other labels for this service/router
+  traefik.http.routers.owncloud.middlewares: owncloud-oidc-rewrite
+  traefik.http.middlewares.owncloud-oidc-rewrite.replacepathregex.regex: ^\.well-known/openid-configuration
+  traefik.http.middlewares.owncloud-oidc-rewrite.replacepathregex.replacement: index.php/apps/openidconnect/config
+```
+
+##### Owncloud OIDC Plugin
+
+Enable the OIDC plugin in your Owncloud installation via the _Market_.
+
+The OIDC plugin cannot be configured from the Owncloud UI, it must be done via the either the `config.php` file or by
+storing the configuration in the Owncloud database.
+Depending on how your Owncloud deployment is setup, the `config.php` file might be exposed in different places.
```

Comment:
I do not know all of the possible places. If not using Docker, it will depend entirely on their setup. I'll add a note about referring to the setup guide for their chosen deployment method to find the file.